### PR TITLE
Avi/language server/vs code client

### PR DESCRIPTION
**Pull request by Avi-D-coder**

### CODE REVIEW

File: lsp/README.md [hunk 1]
```diff
@@ -46,4 +54,32 @@ The VS Code extension offers three configuration options:
 
 - `"nls.server.path"`: Path to nickel language server
 - `"nls.server.trace"`: "Enables performance tracing to the given file"
-- `"nls.server.debugLog"`: "Logs the communication between VS Code and the language server."
```
Comment (ysndr) [inline lsp/README.md:49]: Why should we remove that line?

File: lsp/README.md [hunk 2]
```diff
@@ -46,4 +54,32 @@ The VS Code extension offers three configuration options:
...
+```
+
+### Neo(Vim) with Coc.nvim
+
+Add the `nickel_ls` `JSON` object to your `:CocConfig`/`coc-settings.json`.
+```
+{
+  "languageserver": {
+    "nickel_ls": {
+      "command": "nls",
```
Comment (ysndr) [inline lsp/README.md:75]: Can you add a comment, that `command` would be `nls --trace <file>` to enable performance tracing?

File: lsp/README.md [hunk 3]
```diff
@@ -18,6 +18,14 @@ To enter a shell with `nickel` and `nls` run
 nix shell github:tweag/nickel
 ```
 
+Alternatively to install both using nix run:
+
+```
+git clone https://github.com/tweag/nickel.git
+cd nickel
+nix-env -f . -i
```
Comment (ysndr) [inline lsp/README.md:26]: Should we rather endorse the new nix command `nix build` or mention that this is the method to install without flakes?

**Review by ysndr — APPROVED** [review]
Thank you, generally looking good, just left some comments comments

File: lsp/README.md (hunk 3 above)
Comment (Avi-D-coder) [reply lsp/README.md:26]: I have no preference.

File: lsp/README.md (hunk 1 above)
Comment (Avi-D-coder) [reply lsp/README.md:49]: Good catch. That was a mistake.

File: lsp/README.md (hunk 3 above)
Comment (ysndr) [reply lsp/README.md:26]: We should at least mention `nix build` or `nix profile install` as they are turning into the standard commands for nix

Comment (Avi-D-coder) [reply lsp/README.md:39]: Added

**Review by ysndr — APPROVED** [review]

**Pull request merged**